### ISSUE TITLE
Validation accuracy

### ISSUE THREAD

**Issue by gerardsmith** [runnanchen/Anatomic-Landmark-Detection#2]
Hi @runnanchen 
Fascinating repository and article, I enjoyed reading about it and exploring your implementation.
I have successfully run your code with the same training and validation csv's, however, got quite a significant discrepancy between training and validation performance, where training is a lot higher indicating the model is overfitting. This is not too surprising given the small dataset size, but doesn't quite match the performance published, even when applying augmentation. Is there something additional that you implemented to improve performance to the level that you published? I would love to improve by validation performance so any advice would be greatly appreciated.

**Comment by runnanchen** [comment]
The performance is reproducible before the code release. Maybe you could use our processed data and try again. Thanks
